pkg/kotsadmconfig/validation/types.go

```diff
@@ -0,0 +1,24 @@
+package validation
+
+import (
+	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
+	"github.com/replicatedhq/kots/kotskinds/multitype"
+)
+
+type ConfigGroupValidationError struct {
+	Name       string                      `json:"name"`
+	Title      string                      `json:"title"`
+	ItemErrors []ConfigItemValidationError `json:"item_errors"`
+}
+
+type ConfigItemValidationError struct {
+	Name             string                 `json:"name"`
+	Type             string                 `json:"type"`
+	Value            multitype.BoolOrString `json:"value"`
+	ValidationErrors []ValidationError      `json:"validation_errors"`
+}
+
+type ValidationError struct {
```

Comment:
This whole type is not needed IMO. Just return a new error with the message defined by the vendor.

Reply:
like mentioned here, https://github.com/replicatedhq/kots/pull/3800#discussion_r1164687355
+ adding the details would help for user to understand when using cli `kubectl kots set config`